internal/workflow/processing.go

```diff
@@ -809,6 +809,40 @@ func (w *ProcessingWorkflow) transferAM(sessCtx temporalsdk_workflow.Context, ti
 	// Set AIP "stored at" time.
 	tinfo.StoredAt = temporalsdk_workflow.Now(sessCtx).UTC()
 
+	// Set package location
+	{
+		ctx := withLocalActivityOpts(sessCtx)
+		err := temporalsdk_workflow.ExecuteLocalActivity(
+			ctx,
+			setLocationIDLocalActivity,
+			w.pkgsvc,
+			tinfo.req.PackageID,
+			uuid.MustParse(w.cfg.AM.AMSSLocationID),
```

Comment:
Brave.

Reply:
Yeah, that's a bad panic waiting to happen. :sweat:

Reply:
What would Viper do if you define AMSSLocationID as a uuid.UUID? If undefined, you can always check for `uuid.Nil`.

Reply:
Oh forget it, you'd need to pass a custom decoder func to viper, see https://pkg.go.dev/github.com/spf13/viper#DecoderConfigOption.

Reply:
I thought you were being brave because that config already passed some validation. If not, we shouldn't be that brave ;)

Reply:
Look, found one!

https://github.com/go-saas/kit/blob/8e55a6f58fa1e5f3ae8d7aeff025b38f8fed8a93/pkg/mapstructure/mapstructure.go#L10-L21

Reply:
Okay, I implemented https://github.com/go-saas/kit/blob/8e55a6f58fa1e5f3ae8d7aeff025b38f8fed8a93/pkg/mapstructure/mapstructure.go#L10-L21 with viper to allow decoding AMSSLocationID to a `uuid.UUID`.  It definitely simplifies the type checking inside the app. :)

It was also a good reminder that viper decodes comma-separated strings to a slice by default. It may be better if I  update https://github.com/artefactual-sdps/temporal-activities/blob/main/removefiles/activity.go#L20 to a slice instead of string. :thinking: